Source/Fuse.Common/Timer.uno

```diff
@@ -41,23 +41,30 @@ namespace Fuse
 			if (time > _interval)
 			{
 				_callback();
-
-				if (_once) Stop();
+				if (_once) Dispose();
 				else _startTime = now;
 			}
 		}
 
+		public void Dispose()
+		{
+			if(!_running)
+				return;
+			Stop();
```

Comment:
Could you do:
```uno
if (_running) 
	Stop();
```
instead? "If-not-running-return-else-stop" reads kind of worse, and is more awkward to extend in the future...